pkg/security/security.go

```diff
@@ -344,3 +348,23 @@ func ExtractRequestToken(req *http.Request) (string, error) {
 
 	return "", fmt.Errorf("no bearer token exists in HTTP authorization header")
 }
+
+func GetOSRootPath() string {
+	// Get and store the OS CA certificate path
```

Comment:
point to where this exists in go, https://golang.org/src/crypto/x509/root_linux.go